validator/lib/validateB3dm.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+var Cesium = require('cesium');
+var DeveloperError = Cesium.DeveloperError;
+module.exports = validateB3dm;
+
+/**
+ * Test if the provided data is gzipped.
+ *
+ * @param {Buffer} content A buffer containing the contents of a b3dm tileset.
+ * @returns {Boolean} True if the content is valid according to the specs:
+ * https://github.com/AnalyticalGraphicsInc/3d-tiles/tree/master/TileFormats/Batched3DModel, False if not.
```

Comment:
Use @link.

For example: `{@link http://www.khronos.org/registry/webgl/specs/latest/#5.2|WebGLContextAttributes}`

Also run `npm run jsDoc` to verify that the documentation looks good.